test/kinematic/vector/linear_displacement_is_angular_displacement_cross_radius_test.py

```diff
@@ -0,0 +1,77 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+from sympy import S
+from symplyphysics import (
+    errors,
+    units,
+    convert_to,
+    Quantity,
+    SI,
+    QuantityVector,
+)
+from symplyphysics.laws.kinematic.vector import (
+    linear_displacement_is_angular_displacement_cross_radius as linear_displacement_law,
+)
+
+# Description
+## A body is rotating about a fixes axis. It makes a rotation of (0, 0, 1e-5) rad, small enough so
+## that its radius vector can be considered constant and equal to (0, 0.1, 0) m. During this rotation
+## the body's linear displacement amounts to (-1e-6, 0, 0) m.
+
+
+@fixture(name="test_args")
+def test_args_fixture():
+    theta = QuantityVector([0.0, 0.0, S(10)**(-5)])
```

Comment:
`1e-5` doesn't work?

Reply:
No; pytest doesn't like something and produces `1e-6 != 1e-6 ± 1e-7` as a result if I don't use sympy's rationals.

Reply:
Let me check

Reply:
I fixed it for you.